std/process.d

```diff
@@ -2046,6 +2113,31 @@ enum Config
     stderrPassThrough = 128,
 }
 
+/**
+A sturct that controls the behavior of process creation functions in this module.
+*/
+struct AdvancedConfig
+{
+    /**
+    A function type that notifies error in forked process.
+    */
+    alias AbortOnError = void delegate() nothrow @nogc @safe;
+
+    /**
+    A function that is called before `exec` in $(LREF spawnProcess).
+    $(LREF AbortOnError) can be called to notify errors in `preExecFunction` and
+    to abort forked process.  On Windows, this member has no effect.
+    */
+    void function(AbortOnError) nothrow @nogc @safe preExecFunction;
```

Comment:
You can use `version($OS)` to include or exclude this function, depending on the platform. In this case, you can have something like:

```d
version(Windows) {}
else { /* define function here */}
```

Reply:
And, you can use `version (StdDdoc)` to write definitions of symbols that will be used only for the documentation, so that the rendered documentation is the same no matter what OS it is rendered on.

Reply:
Thank you for the suggestion.
I move to the definitions to `version (Posix)` and `version (StdDdoc)` blocks.